packages/remote-config/e2e/config.e2e.js

```diff
@@ -697,8 +697,8 @@ describe('remoteConfig()', function () {
         (await isSupported()).should.equal(true);
       });
     });
-
-    describe('onConfigUpdate', function () {
+    // SKIPPED: Skipping suite of onConfigUpdate tests until they're fixed
+    describe.skip('onConfigUpdate', function () {
```

Comment:
I cherry-picked this commit to a clean branch for separate PR to get it in now for all PRs until fixed, once it merges to main for everyone's benefit you'll want to rebase it out over on this PR. Cool feature, but such a hassle.